src/resonanceReconstruction/rmatrix/legacy/unresolved/CompoundSystem/src/evaluate.hpp

```diff
@@ -0,0 +1,47 @@
+/**
+ *  @brief Evaluate the cross sections at the given energy
+ *
+ *  @param[in] energy       the incident energy
+ *  @param[in,out] result   a map containing the accumulated cross sections
+ */
+void evaluate( const Energy& energy,
+               std::map< ReactionID, CrossSection >& result ) {
+
+// ----- DEBUG -----
+std::cout << "-- CompoundSystem is working on this energy: " << energy << " --" << std::endl;
+// ----- DEBUG -----
```

Comment:
Does this need to be removed?

Reply:
Once the tests in RECONR show us that it works as advertised, yes. That's why I put in that first and last line of comment. These also only appear at the beginning or end of a function.